cli.py

```diff
@@ -149,10 +149,10 @@ def start(agent_name: str, no_setup: bool):
             setup_process.wait()
             click.echo()
 
-        subprocess.Popen(["./run_benchmark", "serve"], cwd=agent_dir)
-        click.echo("⌛ (Re)starting benchmark server...")
-        wait_until_conn_ready(8080)
-        click.echo()
+        # subprocess.Popen(["./run_benchmark", "serve"], cwd=agent_dir)
+        # click.echo("⌛ (Re)starting benchmark server...")
+        # wait_until_conn_ready(8080)
+        # click.echo()
```

Comment:
Doesn't work: `Command not found: agbenchmark`

Reply:
:? we should open an issuee to fix this :(

Reply:
nit:
```diff
+        # FIXME: Doesn't work: Command not found: agbenchmark
```

Reply:
Added comment and Linear issue